src/main/resources/templates/datewarehouse/etl-jobs-config.json

```diff
@@ -9,37 +9,142 @@
-				"client": "string",
-				"client_version": "string",
-				"entity_id": "bigint"
-			},
-			"partitionKeys": {
-				"record_date": "date"
+			"description" : "The table contain access records. Each record reflects a single API request received by Synapse server. The record data is useful for audit and to analyse API performance such as delays, error or success rate.",
+			"columns":[{
+				"name": "session_id",
+				"type" : "string",
+				"comment" : "A unique number that Synapse assigns for the duration of a session. Sessions are linked to a user or API key or token."
 			},
+			{
+				"name": "timestamp",
+				"type" : "timestamp",
+				"comment" : "The timestamp when the user sent a request to Synapse server."
+			},
+			{
+				"name": "user_id",
+				"type" : "bigint",
+				"comment" : "The unique identifier for the Synapse user."
+			},
+			{
+				"name": "method",
+				"type" : "string",
+				"comment" : "The http method for the API call."
+			},
+			{
+				"name": "request_url",
+				"type" : "string",
+				"comment" : "The URL of the request."
+			},
+			{
+				"name": "user_agent",
+				"type" : "string",
+				"comment" : "The User-Agent header from the http request. see: https://en.wikipedia.org/wiki/User-Agent_header"
+			},
+			{
+				"name": "host",
+				"type" : "string",
+				"comment" : "The IP address of the host that made the request."
+			},
+			{
+				"name": "origin",
+				"type" : "string",
+				"comment" : "The host name of the portal making request, e.g., https://staging.synapse.org, https://adknowledgeportal.synapse.org, https://dhealth.synapse.org."
+			},
+			{
+				"name": "x_forwarded_for",
+				"type" : "string",
+				"comment" : "The HTTP Header contains IP address of user connecting through a proxy. see: https://en.wikipedia.org/wiki/X-Forwarded-For"
+			},
+			{
+				"name": "via",
+				"type" : "string",
+				"comment" : "The HTTP Header Via informs the server of proxies through which the request was sent."
+			},
+			{
+				"name": "thread_id",
+				"type" : "bigint",
+				"comment" : "The identifier of the thread in which the request was processed."
+			},
+			{
+				"name": "elapse_ms",
+				"type" : "bigint",
+				"comment" : "The total time consumed processing the user request in milliseconds."
+			},
+			{
+				"name": "success",
+				"type" : "boolean",
+				"comment" : "Indicates if the user request succeeded (true) or failed (false)."
+			},
+			{
+				"name": "stack",
+				"type" : "string",
+				"comment" : "The stack (prod, dev) on which request was made."
+			},
+			{
+				"name": "instance",
+				"type" : "string",
+				"comment" : "The version of the stack that processed the request."
+			},
+			{
+				"name": "vm_id",
+				"type" : "string",
+				"comment" : "The unique identifier of the actual EC2 instance that processed the request."
+			},
+			{
+				"name": "return_object_id",
+				"type" : "string",
+				"comment" : "The Synapse id of the object affected by the request, if available."
```

Comment:
What does "affected" mean?  Updated?  Created? Downloaded?

Reply:
@SandhraSokhal As a reminder, the requested clarification should go into the .json file, not just into GitHub.  In this case you might replace "affected by the request, if available" with "referenced in the request, if any"